### ISSUE TITLE
JS/XSS Injection via execCommand in PasteUtils

### ISSUE THREAD

**Issue by akhil-reni** [OvidijusParsiunas/deep-chat#95]
### Vulnerable code

```
export class PasteUtils {
  public static sanitizePastedTextContent(event: ClipboardEvent) {
    event.preventDefault();
    const text = event.clipboardData?.getData('text/plain');
    document.execCommand?.('insertHTML', false, text);
  }
}

```

### Description
A potential security vulnerability exists in the `PasteUtils` class, specifically in the `sanitizePastedTextContent` method. The current implementation uses `document.execCommand('insertHTML', false, text)`, which can allow JavaScript injection through pasted content.

### Steps to Reproduce
1. Copy a string that includes JavaScript code.
2. Paste it into the relevant field where `PasteUtils.sanitizePastedTextContent` is used.
3. The JavaScript code gets executed.

### Expected Behavior
Pasted text should be sanitized to prevent any potential JavaScript execution.

### Actual Behavior
JavaScript can be executed through the `execCommand('insertHTML', ...)` method, leading to potential security risks.

### Suggested Fix
Replace `document.execCommand('insertHTML', false, text)` with `document.execCommand('insertText', false, text)`. This change will insert the text as plain text, mitigating the risk of JavaScript execution.

### Additional Information
<img width="1084" alt="Screenshot 2024-01-11 at 12 29 53 PM" src="https://github.com/OvidijusParsiunas/deep-chat/assets/36848653/58c975b3-067b-43fe-8d01-d38fa303603e">

**Comment by OvidijusParsiunas** [comment]
You're an absolute star for raising this @akhil-reni.
I have included this change in the following [commit](https://github.com/OvidijusParsiunas/deep-chat/commit/0cc96ed7218cc6e67a185ed4fdf0f111db8c619c). This should hopefully be included in the main release tomorrow. Thanks!

**Comment by OvidijusParsiunas** [comment]
This has now been released in version `1.4.7`. Thankyou!
